rag_experiment_accelerator/doc_loader/documentLoader.py

```diff
@@ -75,6 +70,7 @@ def load_documents(
         FileNotFoundError: When the specified folder does not exist.
     """
 
+    # ['*'] - to allow any supported format.
     if "*" in allowed_formats:
         allowed_formats = _FORMAT_VERSIONS.keys()
 
```

Comment:
Severity: Major - The removal of `chunk_size`, `overlap_size`, and `azure_document_intelligence_model` parameters seems to reduce functionality significantly without providing an alternative. Consider documenting what will now happen in their absence or whether the function's behavior has changed fundamentally. Loss of these parameters could lead to unoptimized document processing as users no longer have control over chunking conditions. If the design is pivoting, ensure that there’s a strong justification.